master/master.cfg

```diff
@@ -22,14 +21,14 @@ from buildbot.schedulers.forcesched import (ForceScheduler, FixedParameter,
                                             StringParameter, UserNameParameter)
 from buildbot.process import factory
 from buildbot.status import html, mail, words
-from buildbot.steps.shell import (ShellCommand, Configure, Compile,
-                                  Test as BaseTest)
+from buildbot.steps.shell import ShellCommand, Configure, Compile
 from buildbot.steps.transfer import DirectoryUpload
 from buildbot import util, locks
 from buildbot.steps.source.git import Git
 
 sys.path.append(os.path.dirname(__file__))   # noqa: E402
 
+from custom import steps as custom_steps
```

Comment:
I would suggest to implement classes rather than the module here, but it's just a matter of coding style, it's up to you ;-)